packages/phrases/src/locales/en/translation/admin-console/sign-in-exp/content.ts

```diff
@@ -0,0 +1,48 @@
+const content = {
```

Comment:
this are all just file renaming, not sure why git didn't recognize it.

Reply:
It's OK.